pkg/drivers/kic/oci/cli_runner.go

```diff
@@ -32,6 +33,9 @@ import (
 	"k8s.io/minikube/pkg/minikube/style"
 )
 
+var Lock sync.Mutex
+var WarningPrintedSet = make(map[string]bool)
```

Comment:
do these variables neeed to be public ? if no let them private
also add comment
consider better name

alreadyWarnedCmds

and the other one better name maybe,
warnLock ?